spec/models/admin_dashboard_data_spec.rb

```diff
@@ -192,6 +192,52 @@ def my_test_method
     end
   end
 
+  describe 'pwa_config_check' do
+    subject { described_class.new.pwa_config_check }
+
+    it 'alerts for large_icon missing' do
+      SiteSetting.large_icon = nil
+      expect(subject).to_not be_nil
```

Comment:
Perhaps we can test for the exact object here? `to_not be_nil` is too generic and returning an incorrect object will cause this test to pass.